config/jobs/kubernetes-sigs/cluster-api-boostrap-provider-kubeadm/cluster-api-bootstrap-provider-kubeadm-presubmits.yaml

```diff
@@ -4,12 +4,15 @@ presubmits:
   - name: pull-cluster-api-bootstrap-provider-kubeadm-verify
     path_alias: "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm"
     always_run: true
+    optional: false
```

Comment:
i think `always_run` will override `optional` so it's not needed.

Reply:
Ah good to know

Reply:
Is that the case? or does it potentially change the default?

I hesitate because I'm worried we might make a currently required job optional and non-blocking.